biblibre/urungi
# Enable HTML5 mode and allows Urungi to be served under a subdirectory

Test plan:
1. Change the reverse proxy settings so that urungi is served under a
   subdirectory. For example with Nginx:

```
   server {
       location /urungi/ {
           proxy_pass http://localhost:8080;
       }
   }
```

2. Edit your local config file (eg. config/local-development.js) and set
   the `base` config key
```
   {
       base: '/urungi',
   }
```

3. Restart the node server
4. In the browser go to http://yourdomain.com/urungi/ and verify that
   everything works.
   Notice that the '/#/' in the URL is gone. For example,
   http://yourdomain.com/urungi/#/reports is now
   http://yourdomain.com/urungi/reports

Closes #139
